app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -169,9 +168,6 @@ def copy_repos(new_repo, new_version, content, dep_solve)
             copy_output += copy_yum_content(new_repo, solve_dependencies,
                                             content[:package_ids],
                                             content[:errata_ids])
-
-            plan_action(Katello::Repository::MetadataGenerate, new_repo)
```

Comment:
Why are these removed here? I believe they're still needed for types other than `:pulp3_yum`.

Reply:
In 4.0, there was no call to `copy_repos`, but I can see why they would still be needed there. 

The original 9441 was picked, but there was a dependent PR that was the "pulp2 action removal" work, which never made it into 4.0.

Reply:
Does it not work to make it look like it does in master right now? I realized also that `copy_deb_content` would bug out if there are Pulp 3 deb repos to be copied.

Reply:
No because master has diverged sufficiently from 4.0 the change set can't be the same. That was the problem with the CP as it is.